src/main/kotlin/ink/abb/pogo/scraper/Main.kt

```diff
@@ -93,7 +97,12 @@ fun main(args: Array<String>) {
     Log.normal("Logged in as $username with token ${auth.token.contents}")
 
     if (token.isBlank()) {
-        Log.normal("Set this token in your config to log in directly")
+        Log.normal("Setting this token in your config")
+        settings.setToken(auth.token.contents)
+        val out = FileOutputStream("config.properties")
+        settings.properties.store(out,null)
```

Comment:
:/ Then I'd rather not have this in here because it will probably confuse people